### ISSUE TITLE
[Bug] Monaco "vs" theme for SQL language does not meet minimum color contrast issue threshold for .mtk4 and mtk18

### ISSUE THREAD

**Issue by shubhamPPrakash** [microsoft/monaco-editor#5461]
### Reproducible in vscode.dev or in VS Code Desktop?

- [x] Not reproducible in [vscode.dev](https://vscode.dev) or VS Code Desktop

### Reproducible in the monaco editor playground?

- [ ] Not reproducible in [the monaco editor playground](https://microsoft.github.io/monaco-editor/playground.html)

### Monaco Editor Playground Link

https://microsoft.github.io/monaco-editor/playground.html?source=v0.55.1#XQAAAAJoAQAAAAAAAABBqQkHQ5NjdMjwa-jY7SIQ9S7DNlzs5W-mwj0fe1ZCDRFc9ws9XQE0SJE1jc2VKxhaLFIw9vEWSxW3yscwzblk-_1uhs0-3YGTBOsVJ4S4FwfCDQb5-2uUHXTVAefbVSd9IzaZ_vv9VR9KugsKiiNFGo32AmK1qWwDpnVOw0wveTSJD_3iO4BKLEw9tppV76hyOiRWV1v9unOkf_vqEKcuw4cacr8Pfxvkt3UCKw3aCszRSkppL-1wKEuMbjhVRIe7vr73CAQ6_q7-YsWc1rjqrkR1l8IJtiT0vbyr2iClBxBIOEv_As8KnDEPRwUloUrBImqABZ6_1NiMOAB46XoPH_m2CVfKgDDZpKg3SC1X8X1QAnOk5Yy4BI7lp4Mq59qUoNYOO-L70fuIqgfkJ__lDLFl

### Monaco Editor Playground Code

```typescript
const value = /* set from `myEditor.getModel()`: */ `SELECT * FROM DB
WHERE ENV = 'TEST'
AND IS_VERIFIED IS TRUE

`;

// Hover on each property to see its docs!
const myEditor = monaco.editor.create(document.getElementById("container"), {
	value,
	language: "sql",
	automaticLayout: true,
});
```

### Reproduction Steps

Open Monaco Playground
Set the language as sql
Write the sql query which includes 'SOME TEXT' and IS
Use some Accessibility tool for testing the contrast ratio like AXE-CORE

<img width="1029" height="530" alt="Image" src="https://github.com/user-attachments/assets/ed95f412-9e8a-4bd8-a0e2-25c4111b42f5" />

### Actual (Problematic) Behavior

The contrast ration for some token with classes such .mtk4 and .mtk18 will have contrast ratio below WCAG 2 AA minimum contrast ratio threshold which is 4.5:1

### Expected Behavior

The Contrast ration for all token should be more than 4.5:1

### Additional Context

https://dequeuniversity.com/rules/axe/4.12/color-contrast?application=AxeChrome

**Comment by rish0000-dot** [comment]
@vs-code-engineering
/assign
